spec/glimr_api_client/api_spec.rb

```diff
@@ -1,8 +1,36 @@
 require 'rails_helper'
-require 'support/shared_examples_for_api_calls'
 
 RSpec.describe GlimrApiClient::Api, '#post' do
-  include_examples 'anonymous object'
+  let(:docpath) { '/Live_API/api/tdsapi' }
+  let(:api_endpoint) { 'endpoint' }
+  let(:path) { [docpath, api_endpoint].join('/') }
+
+  # Required for run/build ordering in mutation tests.
+  let(:object) do
```

Comment:
I see that's been like that before – but it looks a bit confusing to my fresh eyes!